research/2019-12-16_searching_with_the_calm_api.md

```diff
@@ -0,0 +1,241 @@
+# Searching with the CALM API
+
+### 16 December 2019, AWLC
+
+This has some notes on using the CALM API, and an example script for searching all the records in the CALM catalogue.
+
+
+
+## Motivation
+
+As part of the born-digital content migration, I had a collection of directories with accession-like numbers, and I wanted to get a list of all the accession numbers in CALM, so I could match the directories to accession numbers:
+
+    AAUICW_2185 ~> AAUICW/2185 ?
+    AAUNAM_2213 ~> AAUNAM/2213 ?
+    AAUNAM_2254 ~> AAUNAM/2254 ?
+
+Exporting a list of accession numbers from the CALM GUI would have been tricky (I don't have access to CALM on my Mac), so I tried using the CALM API to export the data.
+
+<details>
+  <summary>What is an accession/accession number?</summary>
+
+  > In library science, an *accession* is all the material that gets collected at one time from a single source.
```

Comment:
I'd recommend saying 'In archives practice' here. Libraries tend to use 'acquisition' instead, which is subtly different to accession.